plugins/modules/efs.py

```diff
@@ -79,6 +79,12 @@
         description:
             - If the throughput_mode is provisioned, select the amount of throughput to provisioned in Mibps.
         type: float
+    backup:
+        description:
+            - Changes the file system backup policy. Use this option to start or stop automatic backups of the file system.
+            - By default, automatic backup of the file system is disabled.
+        choices: ['enabled', 'disabled']
```

Comment:
There is more choices available: 'ENABLED'|'ENABLING'|'DISABLED'|'DISABLING'

See: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/efs.html#EFS.Client.put_backup_policy

Reply:
@goneri , thank you for the detailed review. 
For this part, I intentionally left out the other two choices (ENABLING & DISABLING) since when any of the two is passed as a parameter, the error `BadRequest` is returned. The message of the error states that only `ENABLED` or `DISABLED` should be specified. The error message below:

```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: botocore.errorfactory.BadRequest: An error occurred (BadRequest) when calling the PutBackupPolicy operation: PutBackupPolicy must specify either 'ENABLED' or 'DISABLED'

``` 

So it seemed to me to include them as choices would only confuse users since their use would result in an error. 

Any thoughts on this situation?